sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -337,6 +363,46 @@ export type CharFilter = MappingCharFilter | PatternReplaceCharFilter;
 // @public
 export type CharFilterName = string;
 
+// @public
+export type ChatCompletionExtraParametersBehavior = string;
+
+// @public
+export interface ChatCompletionResponseFormat {
+    chatCompletionSchemaProperties?: ChatCompletionResponseFormatJsonSchemaProperties;
+    type?: ChatCompletionResponseFormatType;
+}
+
+// @public
+export interface ChatCompletionResponseFormatJsonSchemaProperties {
+    description?: string;
+    name?: string;
+    schema?: ChatCompletionSchema;
+    strict?: boolean;
+}
+
+// @public
+export type ChatCompletionResponseFormatType = string;
+
+// @public
+export interface ChatCompletionSchema {
+    additionalProperties?: boolean;
+    properties?: string;
+    required?: string[];
+    type?: string;
+}
+
+// @public
+export interface ChatCompletionSkill extends WebApiSkill {
+    apiKey?: string;
+    commonModelParameters?: CommonModelParameters;
+    extraParameters?: {
+        [propertyName: string]: any;
```

Comment:
If we can use `unknown` instead of `any` that would be lovely.

Reply:
Done!